hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoGetAccountInfoHandler.java

```diff
@@ -261,7 +261,7 @@ private String getContractAccountId(Account account, final Bytes alias) {
         }
         // If we can recover an Ethereum EOA address from the account key, we should return that
         final var evmAddress = tryAddressRecovery(account.key(), EthSigsUtils::recoverAddressFromPubKey);
-        if (evmAddress != null && evmAddress.length == EVM_ADDRESS_LEN) {
+        if (evmAddress.length == EVM_ADDRESS_LEN) {
```

Comment:
It is possible to return null from `tryAddressRecovery` right ? If so we will throw NPE here